docs/guide/execute-transaction.md

```diff
@@ -52,6 +53,27 @@ Examples of parameter [`ExecParams`](https://algobuilder.dev/api/algob/modules/r
   }
 ```
 
+To deploy an ASA without using `asa.yaml`:
+
+```js 
+  {
+    type: types.TransactionType.DeployASA,
+    sign: types.SignType.SecretKey,
+    fromAccount: john.account,
+    asaName: 'silver-12',
+    asaDef: {
+      total: 10000,
+      decimals: 0,
+      defaultFrozen: false,
+      unitName: "SLV",
+      url: "url",
+      metadataHash: "12312442142141241244444411111133",
+      note: "note"
+    },
+    payFlags: {}
+  };
```

Comment:
shall we add a function call?

Reply:
In this docs we have specified transaction parameters to pass to executeTransaction.

Reply:
OK